env.config.js

```diff
@@ -0,0 +1,51 @@
+const path = require('path');
```

Comment:
Could you rename this file(e.g. webpack.env.config)? it made me confused because there is also can be another.env file with ENV VARIABLES